src/test/java/com/bookstore/controller/ShoppingCartControllerTest.java

```diff
@@ -0,0 +1,182 @@
+    @Sql(scripts = ADD_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = DELETE_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    void createCartItem_validRequest_isOk() throws Exception {
+        String token = jwtUtil.generateToken("user@example.com");
+
+        ShoppingCartResponseDto expected = buildValidCartResponseDto(VALID_ID);
+        MvcResult result = mockMvc.perform(post("/cart")
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                        .content(objectMapper.writeValueAsString(buildValidItemRequestDto()))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+        ShoppingCartResponseDto actual = objectMapper.readValue(result.getResponse()
+                .getContentAsString(), ShoppingCartResponseDto.class);
+
+        assertNotNull(actual);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("Valid case of the updateItemQuantity method run with valid request")
+    @Sql(scripts = ADD_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = DELETE_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    void updateItemQuantity_validRequest_isOk() throws Exception {
+        String token = jwtUtil.generateToken("user@example.com");
+
+        CartItemResponseDto expected = buildValidItemResponseDto(VALID_ID);
+        MvcResult result = mockMvc.perform(put("/cart/cart-items/" + VALID_ID)
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                        .content(objectMapper.writeValueAsString(
+                                buildCartItemRequestDtoWithoutBookId()))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+        CartItemResponseDto actual = objectMapper.readValue(result.getResponse()
+                .getContentAsString(), CartItemResponseDto.class);
+
+        assertNotNull(actual);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("Valid case of the deleteItem method run with valid request")
+    @Sql(scripts = ADD_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = DELETE_USER_CART_AND_CART_ITEMS,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    void deleteItem_validRequest_isOk() throws Exception {
+        String token = jwtUtil.generateToken("user@example.com");
+
+        CartItemResponseDto expected = buildValidItemResponseDto(VALID_ID);
+        MvcResult result = mockMvc.perform(delete("/cart/cart-items/" + VALID_ID)
```

Comment:
It is not a mistake, but we can use it like ⬇️ also
```suggestion
        MvcResult result = mockMvc.perform(delete("/cart/cart-items/{id}", VALID_ID)
```